File: src/inventory/Models/MonitorModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace inventory.Models
{
    public class MonitorModel
    {
        private string _lastUpdate;
         
        public int MonitorId { get; set; }
        public string SerialNumber { get; set; }
        public string Model { get; set; }
        public string ComputerName { get; set; }

        public DateTime? LastUpdate { get; set; }
        /*{
            get //(DateTime? value)
            { return _lastUpdate; }
            set
            {
                if (value != null)
                {
                    _lastUpdate = value.ToString();
                }
                else
                {
                    _lastUpdate = "";
                }
                
            }
        } */

    }
}

File: src/inventory/ClientApp/app/shared/model/monitor/monitor.model.js
System.register([], function (exports_1, context_1) {
    "use strict";
    var MonitorModel;
    var __moduleName = context_1 && context_1.id;
    return {
        setters: [],
        execute: function () {
            MonitorModel = /** @class */ (function () {
                function MonitorModel(monitorId, computerName, serialNumber, model, lastUpdate) {
                    this.monitorId = monitorId;
                    this.computerName = computerName;
                    this.serialNumber = serialNumber;
                    this.model = model;
                    this.lastUpdate = lastUpdate;
                }
                return MonitorModel;
            }());
            exports_1("MonitorModel", MonitorModel);
        }
    };
});
